### PR TITLE
chore: write mapped and raw data to separate columns

### PR DESCRIPTION
**Pull request by albertyfwu**
tested for salesforce -> postgres and hubspot -> mongo

## Test Plan

[Describe test plan here]

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

File: packages/core/destination_writers/mongodb.ts [hunk 1]
```diff
@@ -249,6 +249,7 @@ export class MongoDBDestinationWriter extends BaseDestinationWriter {
               _supaglue_emitted_at: record.emittedAt,
               _supaglue_is_deleted: record.isDeleted,
               _supaglue_raw_data: record.rawData,
+              _supaglue_mapped_data: record.mappedData,
```
Comment (lucasmarshall) [inline packages/core/destination_writers/mongodb.ts:252]: We also need to switch the normalized data to using this, right?

Comment (albertyfwu) [reply packages/core/destination_writers/mongodb.ts:252]: yea you're right

File: packages/core/destination_writers/mongodb.ts [hunk 2]
```diff
@@ -229,13 +229,15 @@ export class MongoDBDestinationWriter extends BaseDestinationWriter {
       inputStream,
       new Transform({
         objectMode: true,
-        transform: (record: NormalizedRawRecord, encoding, callback) => {
+        transform: (record: ObjectRecord, encoding, callback) => {
           // normalized fields are the same as raw fields for most providers, at least for now
           // we really want this to only be the mapped fields
```
Comment (lucasmarshall) [inline packages/core/destination_writers/mongodb.ts:234]:
```suggestion
```

**Review by lucasmarshall — APPROVED** [review]
small nit, otherwise looks good

**Pull request merged**